### PR TITLE
Consume connector definitions in RB

### PR DESCRIPTION
**Pull request by ffazzini**

### CODE REVIEW

File: activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/main/java/org/activiti/services/connectors/behavior/MQServiceTaskBehavior.java [hunk 1]
```diff
@@ -38,45 +46,62 @@
...
+    private final List<ConnectorDefinition> connectorDefinitions;
+    private final ConnectorActionDefinitionFinder connectorActionDefinitionFinder;
+    private final VariablesMatchHelper variablesMatchHelper;
     private final RuntimeBundleInfoAppender runtimeBundleInfoAppender;
 
     public MQServiceTaskBehavior(IntegrationContextManager integrationContextManager,
                                  ApplicationEventPublisher eventPublisher,
                                  ApplicationContext applicationContext,
                                  IntegrationContextBuilder integrationContextBuilder,
+                                 List<ConnectorDefinition> connectorDefinitions,
```
Comment (erdemedeiros) [inline activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/main/java/org/activiti/services/connectors/behavior/MQServiceTaskBehavior.java]: If you pass the connector list directly to the finder (https://github.com/Activiti/Activiti/pull/1984#discussion_r217698778) you can remove it from here as well.

File: activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/main/java/org/activiti/services/connectors/behavior/MQServiceTaskBehavior.java [hunk 2]
```diff
@@ -38,45 +46,62 @@
...
-                                      IntegrationContextEntity integrationContext) {
-        IntegrationRequestImpl integrationRequest = new IntegrationRequestImpl(integrationContextBuilder.from(integrationContext,
-                                                                                                              execution));
+                                    IntegrationContextEntity integrationContext) {
+
+        String implementation = ((ServiceTask) execution.getCurrentFlowElement()).getImplementation();
+
+        Optional<ActionDefinition> actionDefinitionOptional = connectorActionDefinitionFinder.find(implementation, connectorDefinitions);
+
+        IntegrationRequestImpl integrationRequest = actionDefinitionOptional.isPresent() == true ? new IntegrationRequestImpl(integrationContextBuilder.from(integrationContext,
```
Comment (erdemedeiros) [inline activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/main/java/org/activiti/services/connectors/behavior/MQServiceTaskBehavior.java]: Can be simplified to 
```java
 IntegrationRequestImpl integrationRequest = new IntegrationRequestImpl(integrationContextBuilder.from(integrationContext,
                execution, actionDefinitionOptional.orElse(null)));
```

**Review by erdemedeiros — COMMENTED** [review]
Two small details. It looks good to me otherwise.

**Comment by mteodori** [comment]
@ffazzini @salaboy to be reviewed after the merge of process variables, to be scheduled after release, impact is that connectors would work but all variables will be passed to the connector

**Pull request merged**